packages/ember-metal/lib/utils.js

```diff
@@ -258,6 +258,24 @@ var sourceAvailable = (function() {
   return this;
 }).toString().indexOf('return this;') > -1;
 
+function K(){}
+K.__hasSuper = false;
+
+function checkHasSuper(func) {
+  if (sourceAvailable) {
+    func.toString().indexOf('_super') > -1;
+  }
+  return true;
+}
+
+function hasSuper(func) {
+  let hasSuper = func.__hasSuper;
```

Comment:
JSHint is complaining that `hasSuper` is already defined (possibly because that is the name of the function also).